docker/spec/dependabot/docker/file_updater_spec.rb

```diff
@@ -1139,7 +1139,7 @@
   let(:helmfile) do
     Dependabot::DependencyFile.new(
       content: helmfile_body,
-      name: "values.yaml"
+      name: "values.yaml" # TODO not sure if I need to replace the filenames here or if I can override later in a specific test
```

Comment:
note to self: this fixture seems to be for all the helm chart tests, so will likely need to refactor things a bit... but no need to fully test that the other filename permutations have exhaustive descriptions as that is the kind of thing that will slow down CI over time... just ensure some basic sanity checking to ensure the updater is picking it up, and then rely on the existing tests to verify that the updater is doing the right thing once it picks it up...